diff --git a/node_modules/@changesets/parse/dist/changesets-parse.cjs.js b/node_modules/@changesets/parse/dist/changesets-parse.cjs.js
index 53793c5..a5c0aea 100644
--- a/node_modules/@changesets/parse/dist/changesets-parse.cjs.js
+++ b/node_modules/@changesets/parse/dist/changesets-parse.cjs.js
@@ -21,7 +21,7 @@ function parseChangesetFile(contents) {
   let releases;
 
   try {
-    const yamlStuff = yaml__default["default"].safeLoad(roughReleases);
+    const yamlStuff = yaml__default["default"].load(roughReleases);
 
     if (yamlStuff) {
       releases = Object.entries(yamlStuff).map(([name, type]) => ({
diff --git a/node_modules/@changesets/parse/dist/changesets-parse.esm.js b/node_modules/@changesets/parse/dist/changesets-parse.esm.js
index a48e06b..03f2ab0 100644
--- a/node_modules/@changesets/parse/dist/changesets-parse.esm.js
+++ b/node_modules/@changesets/parse/dist/changesets-parse.esm.js
@@ -13,7 +13,7 @@ function parseChangesetFile(contents) {
   let releases;
 
   try {
-    const yamlStuff = yaml.safeLoad(roughReleases);
+    const yamlStuff = yaml.load(roughReleases);
 
     if (yamlStuff) {
       releases = Object.entries(yamlStuff).map(([name, type]) => ({
diff --git a/node_modules/@changesets/parse/src/index.ts b/node_modules/@changesets/parse/src/index.ts
index 04431e6..41263ea 100644
--- a/node_modules/@changesets/parse/src/index.ts
+++ b/node_modules/@changesets/parse/src/index.ts
@@ -19,7 +19,7 @@ export default function parseChangesetFile(contents: string): {
   let releases: Release[];
   try {
     const yamlStuff: { [key: string]: VersionType } =
-      yaml.safeLoad(roughReleases);
+      yaml.load(roughReleases);
     if (yamlStuff) {
       releases = Object.entries(yamlStuff).map(([name, type]) => ({
         name,
